src/Request/uapi-parser.js

```diff
@@ -258,6 +284,7 @@ Parser.prototype.parse = function (xml) {
       if (self.debug > 2) {
         console.log('Error during automatic resolving version');
       }
+      throw new RequestRuntimeError.VersionParsingError(obj, e);
```

Comment:
It's not yet necessary, as we still have fallback version. But I'd like to keep it.
If we decide to keep it, then debug is not needed.